### PR TITLE
Car/api

### PR DESCRIPTION
**Pull request by TheRedPanda17**

### CODE REVIEW

**Review by rj-lee — APPROVED** [review]

File: myning/api/player.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+def sync():
+    player = Player()
+    stats = Stats()
+    url = API_CONFIG["base_url"] + "/players"
+    headers = {"Authorization": API_CONFIG["auth"]}
+    try:
+        current = get_player(player.id)
+        if current:
+            exists = True
+    except requests.HTTPError:
```
[the remark this answers is not in the document]
Comment (TheRedPanda17) [reply myning/api/player.py:19]: I didn't get any code completion for this and I don't see documentation for this.

**Comment by TheRedPanda17** [comment]
@rj-lee Can you try it locally?

File: myning/api/player.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (rj-lee) [reply myning/api/player.py:19]: instead of using `raise_for_status`, you can check the `response.status_code`

**Comment by rj-lee** [comment]
> @rj-lee Can you try it locally?

works great

**Pull request merged**